externs/shaka/player.js

```diff
@@ -1179,10 +1179,10 @@ shaka.extern.xml.Node;
  *   <br>
  *   Defaults to <code>false</code>.
  * @property {number} updatePeriod
- *   Override the minimumUpdatePeriod of the manifest. The value is in second
- *   if the value is greater than the minimumUpdatePeriod, it will update the
- *   manifest less frequently. if you update the value during for a dynamic
- *   manifest, it will directly trigger a new download of the manifest
+ *   Override the minimumUpdatePeriod of the manifest. The value is in seconds.
```

Comment:
I realized much of this was copied from the DASH version, so I updated both.

Reply:
Thanks a lot